app/src/main/java/com/infomaniak/mail/MainApplication.kt

```diff
@@ -266,6 +264,8 @@ open class MainApplication : Application(), ImageLoaderFactory, DefaultLifecycle
         }
 
         override suspend fun getApiToken(): ApiToken? = AccountUtils.currentUser?.apiToken
+
+        override fun getCurrentUserId(): Int = AccountUtils.currentUserId
```

Comment:
This won't work, the value is computed only once when the tokenInterceptorListener is instanciated

Reply:
Actually not